samples/Samples.Server/Startup.cs

```diff
@@ -35,36 +32,29 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<MessageType>();
             services.AddSingleton<MessageInputType>();
 
-            // http
-            services.AddGraphQLHttp();
-
-            // subscriptions
-            services.Configure<ExecutionOptions<ChatSchema>>(options =>
+            services.AddGraphQL(options =>
             {
                 options.EnableMetrics = true;
-                options.ExposeExceptions = true;
-            });
+                options.ExposeExceptions = Environment.IsDevelopment();
+            })
+            .AddWebSockets()
+            .AddDataLoader();
 
-            services.AddSingleton<ITokenListener, TokenListener>();
-            services.AddSingleton<IPostConfigureOptions<ExecutionOptions<ChatSchema>>, AddAuthenticator>();
-            
-            // register default services for web socket. This will also add the protocol handler.
-            services.AddGraphQLWebSocket<ChatSchema>();
             services.AddMvc();
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+        public void Configure(IApplicationBuilder app)
         {
-            if (env.IsDevelopment())
+            if (Environment.IsDevelopment())
                 app.UseDeveloperExceptionPage();
-            
+
             app.UseDefaultFiles();
             app.UseStaticFiles();
 
             app.UseWebSockets();
-            app.UseGraphQLWebSocket<ChatSchema>(new GraphQLWebSocketsOptions());
-            app.UseGraphQLHttp<ChatSchema>(new GraphQLHttpOptions());
+            app.UseGraphQLWebSockets<ChatSchema>("/graphql");
+            app.UseGraphQL<ChatSchema>("/graphql");
             app.UseGraphQLPlayground(new GraphQLPlaygroundOptions()
```

Comment:
Perhaps the path `/graphql` could be made part of `GraphQLOptions`, so that the `GraphQLPlaygroundOptions` does not need it.

Also, for consistency, it would be nice if you could do `UseGraphQLPlayground("ui/playground")`.

Reply:
That's an interesting idea to add the GraphQL endpoint to `GraphQLOptions`. I agree we should make all of the app builder extensions consistent.

Can you open an issue for this discussion since this PR is closed?

Reply:
For now I've raised a PR https://github.com/graphql-dotnet/server/pull/153 to default the `/graphql` path.